Разметка сайта

Kubich13 opened the pull request:
Не уверен, что нужно так делать разметку для слайдера на первом экране

Kubich13 committed:
Разметка сайта

Kubich13 committed:
Delete DLK-project_SN.html

Kubich13 committed:
Revert "Delete DLK-project_SN.html"

This reverts commit ea724afedae0abfa42ec1850b226facdc2ea844a.

SergeyPechinov commented on DLK-project_SN.html:
```diff
@@ -0,0 +1,188 @@
+    </a>
+    <a class="slideshow-control_fourth" href="#" role="button">
+      <img src="/icon.png" alt="button icon">
+    </a>
```
если нет перехода на другую страницу, лучше сделать button

SergeyPechinov commented on DLK-project_SN.html:
```diff
@@ -0,0 +1,188 @@
+  </header>
+  <main class="main">
+  <div class="slideshow-container">
+    <div class="slideshow-item">
```
можно оставить div'ом , но так же можно обернуть в section и сделать заголовок h2-h6 и скрыть его

SergeyPechinov commented on DLK-project_SN.html:
```diff
@@ -0,0 +1,188 @@
+      <button class="description-button"><img src="/arrowleft.png" alt="left arrow"></button>
+      <button class="description-button"><img src="/arrowright.png" alt="right arrow"></button>
+      <p class="description-text">Modern finds for the modern home. Curated goods by unique designers <br> around the world from fashion to furniture for kids and adults.</p>
+    </section>
```
это часть блока, а не целый блок, не стоит делать section

SergeyPechinov commented on DLK-project_SN.html:
```diff
@@ -0,0 +1,188 @@
+        <img src="/ig.png" alt="instagram icon">
+      </a></li>
+    </ul>
+</nav>
```
Это уже не навигация по сайту, поэтому nav тут не нужен

Kubich13 committed:
Изменения согласно комментариям

Слайдер - вместо дива теперь section

Теперь buttons вместо ссылок на переключения картинок на слайдере

класс description теперь в div

класс connect теперь в div
